build-caching-maven-samples/spotless-project/pom.xml

```diff
@@ -0,0 +1,183 @@
+                                    <inputs>
+                                        <fileSets>
+                                            <fileSet>
+                                                <name>baseDir</name>
+                                                <excludes>
+                                                    <exclude>.idea/*</exclude>
+                                                    <exclude>target/*</exclude>
+                                                </excludes>
+                                            </fileSet>
+                                        </fileSets>
+                                        <properties>
+                                            <property>
+                                                <name>applySkip</name>
+                                            </property>
+                                            <property>
+                                                <name>encoding</name>
+                                            </property>
+                                            <property>
+                                                <name>checkSkip</name>
+                                            </property>
+                                            <property>
+                                                <name>filePatterns</name>
+                                            </property>
+                                            <property>
+                                                <name>formats</name>
+                                            </property>
+                                            <property>
+                                                <name>goal</name>
+                                            </property>
+                                            <property>
+                                                <name>setLicenseHeaderYearsFromGitHistory</name>
+                                            </property>
+                                            <property>
+                                                <name>skip</name>
+                                            </property>
+                                            <property>
+                                                <name>ratchetFrom</name>
+                                            </property>
+                                            <property>
+                                                <name>upToDateChecking</name>
+                                            </property>
+                                        </properties>
+                                        <ignoredProperties>
+                                            <ignore>repositorySystemSession</ignore>
+                                            <ignore>repositories</ignore>
+                                            <ignore>project</ignore>
+                                            <ignore>lineEndings</ignore>
```

Comment:
Do you know if this property would affect the outcome of the `check` goal?

Reply:
I'm not sure if is going to be affected